pkgs/development/python-modules/txtai/default.nix

```diff
@@ -0,0 +1,145 @@
+{ lib
+, buildPythonPackage
+, pythonOlder
+, fetchFromGitHub
+, pythonRelaxDepsHook
+# propagated build input
+, faiss
+, torch
+, transformers
+, huggingface-hub
+, numpy
+, pyyaml
+, regex
+# optional-dependencies
+, aiohttp
+, fastapi
+, uvicorn
+# , apache-libcloud
+, rich
+, duckdb
+, pillow
+, networkx
+, python-louvain
+, onnx
+, onnxruntime
+, soundfile
+, scipy
+, ttstokenizer
+, beautifulsoup4
+, nltk
+, pandas
+, tika
+, imagehash
+, timm
+, fasttext
+, sentencepiece
+, accelerate
+, onnxmltools
+, annoy
+, hnswlib
+, scikit-learn
+, sentence-transformers
+, croniter
+, openpyxl
+, requests
+, xmltodict
+# native check inputs
+, unittestCheckHook
+}:
+let
+  version = "5.5.1";
+  api = [ aiohttp fastapi uvicorn ];
+  # cloud = [ apache-libcloud ];
+  console = [ rich ];
+  database = [ duckdb pillow ];
+  graph = [ networkx python-louvain ];
+  model = [ onnx onnxruntime ];
+  pipeline-audio = [ onnx onnxruntime soundfile scipy ttstokenizer ];
+  pipeline-data = [ beautifulsoup4 nltk pandas tika ];
+  pipeline-image = [ imagehash pillow timm ];
+  pipeline-text = [ fasttext sentencepiece ];
+  pipeline-train = [ accelerate onnx onnxmltools onnxruntime ];
+  pipeline = pipeline-audio ++ pipeline-data ++ pipeline-image ++ pipeline-text ++ pipeline-train;
+  similarity = [
```

Comment:
```suggestion
  console = [ rich ];
  
  database = [ duckdb pillow ];
  
  graph = [ networkx python-louvain ];
  
  model = [ onnx onnxruntime ];
  
  pipeline-audio = [ onnx onnxruntime soundfile scipy ttstokenizer ];
  pipeline-data = [ beautifulsoup4 nltk pandas tika ];
  pipeline-image = [ imagehash pillow timm ];
  pipeline-text = [ fasttext sentencepiece ];
  pipeline-train = [ accelerate onnx onnxmltools onnxruntime ];
  pipeline = pipeline-audio ++ pipeline-data ++ pipeline-image ++ pipeline-text ++ pipeline-train;
  
  similarity = [
```

Reply:
You can resolve this if you disagree this is nescesarry, but I still think it would be a huge readability improvement.